### ISSUE TITLE
SA1503 - incorrect warning with using declarations

### ISSUE THREAD

**Issue by trydis** [DotNetAnalyzers/StyleCopAnalyzers#3074]
As mentioned in https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3004

![](https://user-images.githubusercontent.com/446036/68934785-a3560080-0797-11ea-82d5-5ed0a111c25a.png)

**Comment by sharwell** [comment]
I was not able to reproduce this. A test is added in #3076.

**Comment by trydis** [comment]
This reproduces the warning: https://github.com/trydis/analyzers-repro

Rider JetBrains Rider 2019.2.2
macOS 10.15.1

**Comment by sharwell** [comment]
@trydis Thanks for providing the sample repository! I'm not sure what tool is reporting those warnings, but it's not StyleCop Analyzers. I verified both the IDE and command line build scenarios.

**Comment by trydis** [comment]
Really confusing, but I guess it's Rider in some way, thanks for looking into it.

Rider settings:
`Editor -> Code Style -> Enable StyleCop support: Disabled`
`Editor -> Inspection Settings -> Roslyn Analyzers -> Enable Roslyn Analyzers: Enabled`

I'm able to control it using a ruleset file or #pragma:
```
<?xml version="1.0" encoding="utf-8"?>
<RuleSet Name="Test" Description="Code analysis rules" ToolsVersion="15.0">
    <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
        <Rule Id="SA1106" Action="None"/>
        <Rule Id="SA1501" Action="None"/>
        <Rule Id="SA1503" Action="None"/>
    </Rules>
</RuleSet>
```

Mentioning https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3075 for traceability.
